akka-docs/rst/java/remoting-artery.rst

```diff
@@ -441,24 +480,27 @@ Note that the array based methods can be implemented by delegation like this:
 
 .. includecode:: code/docs/actor/ByteBufferSerializerDocTest.java#bytebufserializer-with-manifest
 
+.. _disable-java-serializer-java-artery:
+
 Disabling the Java Serializer
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-It is possible to completely disable Java Serialization for the entire Actor system.
-
-Java serialization is known to be slow and prone to attacks of various kinds - it never was designed for high 
-throughput messaging after all. However it is very convenient to use, thus it remained the default serialization 
-mechanism that Akka used to serialize user messages as well as some of its internal messages in previous versions.
+With Artery it is possible to completely disable Java Serialization for the entire Actor system.
```

Comment:
I don't think this text has to be so specfic for Artery. I think the previous wording was fine.

It is possible to completely disable Java Serialization for the entire Actor system.